pages/nft-book-store/new.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <PageBody class="pb-[40px]">
+  <component :is="wrapperComponent" class="pb-[40px]">
```

Comment:
Please move the whole component into a new component, keep `/page` to always use `PageBody`